nirodents/workflows/brainextraction.py

```diff
@@ -1,39 +1,44 @@
 # emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
 # vi: set ft=python sts=4 ts=4 sw=4 et:
 """Nipype translation of ANTs' workflows."""
-
+import numpy as np
 # general purpose
-from multiprocessing import cpu_count
 from pkg_resources import resource_filename as pkgr_fn
 
 # nipype
 from nipype.pipeline import engine as pe
 from nipype.interfaces import utility as niu
-from nipype.interfaces.ants import N4BiasFieldCorrection, Atropos
+from nipype.interfaces.ants import N4BiasFieldCorrection
 from nipype.interfaces.ants.utils import AI
-from nipype.interfaces.afni import MaskTool
-from nipype.interfaces.io import DataSink
 
 # niworkflows
 from niworkflows.interfaces.ants import ImageMath
 from niworkflows.interfaces.images import RegridToZooms
-from niworkflows.interfaces.nibabel import ApplyMask
+from niworkflows.interfaces.nibabel import ApplyMask, Binarize
 from niworkflows.interfaces.fixes import (
     FixHeaderRegistration as Registration,
     FixHeaderApplyTransforms as ApplyTransforms,
 )
+from niworkflows.interfaces.registration import (
+    SimpleBeforeAfterRPT as SimpleBeforeAfter
+)
 
 from templateflow.api import get as get_template
+from ..utils.filtering import (
+    gaussian_filter as _gauss_filter,
+    truncation as _trunc
+)
 
-LOWRES_ZOOMS = (0.4, 0.4, 0.4)
+LOWRES_ZOOMS = (0.42, 0.42, 0.42)
+HIRES_ZOOMS = (0.1, 0.1, 0.1)
```

Comment:
These are the two sets of resolutions. The lower for `antsAI`, the higher for `antsRegistration`